Kubernetes raspberry pi (#1)
jnduli/blog_jnduli.co.ke: content/pages/about.md

## WHY THE BLOG ##

> I use this to self document the problems I experience while working and

Reviewer: ```suggestion
I use this to self document problems I experience(during work) and
```

the solutions I come up with. I work on a wide range of technologies and
sometimes I find a fix here and there that needs recording for future
reference. I've found this works better than re-reading the original
source of the solution (I think this is becaue I understand my words
better).

Also it works as a place to let my opinions loose. Perhaps on a book

> here, a movie there, some policy, etc.

Reviewer: This is poorly structured. Rethink how to properly express yourself here.

## DISCLAIMER ##
Please keep this in mind while reading the blog:

_Stuff just is. My mental model of how the world works is incomplete. It
has boundaries. I'm constantly relearning stuff. That, there, is the
thing. Knowing that I'm limited in how I perceive stuff. There is
strength in these "limits" though, they enable you to filter noise and

> make decisions. ~ Bonface Munyoki_

Reviewer: NIT: Lolz. So I'm reading a quote of myself, but it feels poorly structured :sweat:
